FitNesseRoot/FitNesse/UserGuide/FitNesseWiki/MarkupLanguageReference/MarkupPicture/content.txt

```diff
@@ -17,3 +17,8 @@ You may also set the width of the image if you wish by using '''!-!img -w nnn-!'
48CEb87FCJoJSa6Ir8KmR31eZY1Wb3o0eMXvLGLhLqHQqI7SSjDRssQ+4av6raQVsgUldAWzF5d/q/O79w91ll+yGdCS1CAxFICxBX13V+5BvdZylv54KybvJn5mNncA22OgQuW00JFBGOlyJPgpKL5srl8W+zgtxaNXlPh0fQ4mz23oYIuphD3htfNPv39O/cTS2ZX46QjFxxcmorWBkYn4J4H0TaYqg8g08Y5jl+TVB9++I9CB2iK6lFhMKsKmGSKz0NBW2bQKQoDNRSONkYVB1EUO6PUDpYHfkhmk3l1gZ1A7yNAX7Xipxdp9TGqqVB8ij/VFFXscLAzxQZFyQDLo0SgV3YxErpx1MLn+3Vc8eDHF/kyc58jVDnez8hw8FOKHy1GsugT4MwDtQZcnhQYDCPVoFF+D4J2FjDU/YZn+4/uBi2BXjaEapoX2h4n9r34PlDwffB/ir/d4cGa3ZHNbPY8Yx3JbFj9w1+3fOhn/4C2O01Rvari9Y8FlTJNw3FCgvtMY5I3GB8wNoA8bGsnnIOgLkemJrP7ntmzg26jdvNePgDlEU9uk+FbLh16clXj487wZeXdxBR7Zz/DCNSrYLYDEuADfCU2/u4L9AXz6P/D/sokmroT2na1w1X6Y0bxtlUNNhXtbrrq6zJiruUH6S58vu89+3sOGHKk49+bVzdoWaimZV2k2RxnwvfQGlfUkwAvsen04JhI02mB3xnSgt+LEy1H6vOuTCs9MK1g4ZsL87at5Glm+DPF3ThNc+IBLRfhs/6485323qwGZ2oO5xma3dkX+qCN3i7uAC0Unue32jiP5LTw8ZsO4/M/Wfv+mtryl1l/CilvxO9ETH3PhYpMXwYTMSK6vTro33CYsf6GFztUN2rdAwTJrTCWO+lt5/HXS7HNqJOEKks+4bsifjowNxi8J4zJdaAULoFvdFnjP79Zq/ejaCnFHHthwkN3V91FFf/iLD9gyO88vFd9ZPro8m9IjdAIi6JCgOBAgYGCI87wtloDTrheA4JjV7IqZAPy0Z3zpr6oBjvMLhyjhU2KAz2qKZtsAM3fAr2LSQ2tvPDcO6a3kfANhGQ1KLX7q9divjEM3bZvuXxYq6pGbduq6dlNgWGi60xBQWbzVJfZXSVHnBWHyxqt+e5g3vZV3NXBnyVqcP42w1QX8IxJpOaDUQzvZQbR9yydcNMCWI8JV2Hqi99I6JiQTWAnujviFSBB7RHXjHSc8TjuyYMRje+rzkxl75mDGriatWsCTKiR5kjLBYGSDn3htwYwSSX0hwuEhMvmqarI3r1uT/bejRUN/vqFb1rEhCMvIpPkNBvp8oFmi0oAuooX6G2iHOzSP6ey5Um5wMhzQNA1VZ0Z+UBLpj+FquaDcVFVYquuLGu07vutjdZ+V2HzVjMDDaIyYOYiMrP42Glv34H702DgCq6Mt+u0ZJcVdG0MdDSAdskOoUX1uWzeqjKb21UOQmtny19m8r1CQjR1Ei48eLE0zgwCxF93xw8f5f34IdJ76JQ+maXtujeqbtSmlYr9aHNkgrKHQkTxJxvVVA8oUW5QSI6C9X7IWX64qvnSzw6B8BfnpsS2NPajvPBAd9Xg53KOtzsp331Pra5kU777yPWCHn4rLvQL2YAEhAaGrOIirr6yDVRHwEbAVehPWMyUaxXiJC7CivCoa9cVIeHhlPh3LOFhlZkFM5HycsIXL6Hw3kNCk8RFAn+OGWMwu5bTye8xq9vImONesGoGYR9sMSpR5ZlsAAADjUlEQVSM0yIjQIzCw9LYMLiwRIbmVw5sHhdt8/00dI0Vk5r+tiLY+NHINMMmp03uNWYpUJquXlBCVi8o1WlYL0mL8dnie8ooZCLNHBRpaRqghRC5/pIRHiyCAIsJ/fsx1h/Mztt47eIyKDgfskOSXjPaxfNI80SmCBEe6K7Ka2LDOYocTWV/7N2sPvzJ+Apcdc2jqEz9diA4LiWBFNHJyu+TLOBgvAOEhlmSx2TCbJIpFlOsNfdJLAgMzkhvNJpDXBIWtCjjc8QPQHwmNdxjvD/kg4zxLtHa38hsagOiImFVeBBi3UUYAmETIzFiRkwSaMzuyhGJPmN/mBUZNyxnUOI4iJQw0EhLNBpC+jQaszYIV7E+cRwax6RZuxgZr7HPkmLVGqxJs2/HTAmTUcjMFIGQd4hFe1B4oOvlvjfzblUUcmZVufbkzBcqppQc1MTQW1+UTpkH5R4ouGdAfa3ZEIF5cnCyf2ItWxt+GDpYXMQWDcFOq6edwEQGF0lLN15vHHgxYcLI+LOiCQ/xGWaCOaTNhFTZvO1lLJpaa3vhneMRHCJ9/vN4aTTQYWwbI8OW6f8w2owlRiSReI9ReBCTYywaQsaGxe1uCakZU7HaQ1Z41Mq44v2I5zwai0RWAmT7UaSdn1tSDu06QfSW53Pa2Ozkwu1rvGfMea0SzSAxYiGqTx4YNP7v32CBYMbTO6AMI4EkanUN/xwElFeBptJYFycTYgf/N6Cu6TV8AKKAjcZYpZnosdj+xwpNRjoEQUuINcGWkFATrhcZoZEGS0pMooKV/y0k3JQVZCItTKizVmEiSBDmqd4jI0wRiId2bnkoGP/z2t2lN+t/BydHrfjr9H0wXgchgq4vzEGFaX7PIda0rniA0VMYzjkzjr04UqhjmFgjCFPLo772Z/7/GSaCHCGlsdaCRh2sW4IW0dpM6riIQkskemqNFlmINFvtR8P9cSHotprxXDkuEEEzyO9XTiTzLTBwDB+biQUECVogKEhwtTHmVEo0rhqBwg2jp9Bd9gk8b1MS6kyhjmEYvCkhUQ8wMs36xDFKC6Le6YmF+mg7Y7qHpIeNAmPHBUKvk4BFghPquGIdU5R0JoEc+W1JTX59M+AaA8ythKF1q6FgOpFfoN6SKPekkEIKKaRQixCFR627CYDhY6z4t3oJAoQKxpjjAi5csIQLXdBlhu6nI3BP3SRVSyGFFFJIQRr/D8F5MBAkaqwxAAAAAElFTkSuQmCC|
```

Comment:
Maybe you can find a bit smaller image? We just want to show it in action, right?

Reply:
I thought about using a minimal image but even an 8 by 8 pixel image would lead to unreadable markup.
So I decided to just use the existing example and added a note addressing the generally bad readability.

Should I use a very minimal image instead (like https://gist.github.com/ondrek/7413434) ?
The rendered image wouldn't be very visible though.